conjur/constants.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+"""
+Constants module
+
+This module holds all constants used across the codebase
+"""
+
+# Builtins
+import os
+
+DEFAULT_CONFIG_FILE = os.path.expanduser(os.path.join('~', '.conjurrc'))
+DEFAULT_NETRC_FILE = os.path.expanduser(os.path.join('~', '.netrc'))
+CERTIFICATE_FILENAME = "conjur-server.pem"
```

Comment:
This is not what I meant. I meant a full path, just like you do for the aforementioned consts.